### ISSUE TITLE
Make Pymathics-graph work again

### ISSUE THREAD

**Issue by rocky** [Mathics3/mathicsscript#23]
```
In[1]:= LoadModule["pymathics.graph"]
Out[1]= pymathics.graph
In[2]:= BinomialTree[3]
```

Used to render something in a matplotlib screen. It no longer does.

**Comment by mmatera** [comment]
It is related to the changes in the way contexts are handled for pymathics. I put `Pymathics` as the default context for pymathics modules, but I put PyMathics in the $ContextPath list. I am fixing this now.

**Comment by rocky** [comment]
@mmatera I wonder if pymathics.graph needs to be told about `formatvalues` now where before it didn't. 

Pymathics uses its own type which imbeds a networkx graph. 
```
In[2]:=  BinomialTree[3]
Traceback (most recent call last):
  File "/home/rocky/.pyenv/versions/3.7.9/bin/mathicsscript", line 11, in <module>
    load_entry_point('mathicsscript', 'console_scripts', 'mathicsscript')()
  File "/home/rocky/.pyenv/versions/3.7.9/lib/python3.7/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/home/rocky/.pyenv/versions/3.7.9/lib/python3.7/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/home/rocky/.pyenv/versions/3.7.9/lib/python3.7/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/rocky/.pyenv/versions/3.7.9/lib/python3.7/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/src/external-vcs/github/Mathics3/mathicsscript/mathicsscript/__main__.py", line 324, in main
    query, source_code = evaluation.parse_feeder_returning_code(shell)
  File "/src/external-vcs/github/mathics/Mathics/mathics/core/evaluation.py", line 278, in parse_feeder_returning_code
    result, source_code = parse_returning_code(self.definitions, feeder)
  File "/src/external-vcs/github/mathics/Mathics/mathics/core/parser/util.py", line 34, in parse_returning_code
    return convert(ast, definitions), source_code
  File "/src/external-vcs/github/mathics/Mathics/mathics/core/parser/convert.py", line 151, in convert
    result = self.do_convert(node)
  File "/src/external-vcs/github/mathics/Mathics/mathics/core/parser/convert.py", line 156, in do_convert
    result = GenericConverter.do_convert(self, node)
  File "/src/external-vcs/github/mathics/Mathics/mathics/core/parser/convert.py", line 24, in do_convert
    head = self.do_convert(node.head)
  File "/src/external-vcs/github/mathics/Mathics/mathics/core/parser/convert.py", line 157, in do_convert
    return getattr(self, '_make_' + result[0])(*result[1:])
  File "/src/external-vcs/github/mathics/Mathics/mathics/core/parser/convert.py", line 163, in _make_Lookup
    value = self.definitions.lookup_name(s)
  File "/src/external-vcs/github/mathics/Mathics/mathics/core/definitions.py", line 397, in lookup_name
    if self.have_definition(n):
  File "/src/external-vcs/github/mathics/Mathics/mathics/core/definitions.py", line 425, in have_definition
    return self.get_definition(name, only_if_exists=True) is not None
  File "/src/external-vcs/github/mathics/Mathics/mathics/core/definitions.py", line 461, in get_definition
    for form, rules in curr.formatvalues.items():
AttributeError: 'BinomialTree' object has no attribute 'formatvalues'
```

The thing that we need to be careful here is that the evaluation phase of core isn't making graphics decisions (SVG, asymptote or whatever).  I imagine there is a way to do this. 

The front-end when it goes to format will see that this is a type the encapsulates a networkx graph and do the right thing there.

**Comment by mmatera** [comment]
What I saw yesterday is that for some reason, BinomialTree and other objects are not loaded in Definitions.  You can check this using MathicsSession objects in a python terminal :
```
from mathics.session  import MathicsSession
session = MathicsSession()
session.evaluate('LoadModule["pymathics.graphs"]')
session.definitions.get_definition("Pymathics`BinomialTree")

session.pymathics
```

**Comment by mmatera** [comment]
As soon as I have time, I will try to fix this.

**Comment by mmatera** [comment]
This PR in Mathics  https://github.com/mathics/Mathics/pull/1152#issue-572973572  comes to fix this issue

**Comment by rocky** [comment]
Fixed in 2.0.0 release
